packager/react-native-xcode.sh

```diff
@@ -41,6 +41,11 @@ elif [[ -x "$(command -v brew)" && -s "$(brew --prefix nvm)/nvm.sh" ]]; then
   . "$(brew --prefix nvm)/nvm.sh"
 fi
 
+# Set up the nodenv node version manager if present
+if [[ -s "$HOME/.nodenv/bin/nodenv" ]]; then
```

Comment:
I don't know enough about nodenv -- do you know if $HOME/.nodenv/bin needs to be added to $PATH for it to work or is this enough?

Also could you change -s to -x?